### DISCUSSION TITLE
Linking third party CSS and JS files

### DISCUSSION THREAD

**Discussion by X3Technology** [vanjarosoftware/Vanjaro.Platform#1458] [Q&A] [answered]
What is the best way to link to a third-party CSS/JS file?  

I would prefer the CSS to be in the HEAD section and the JS to be loaded async at the bottom of the page.

Functionality similar to DNN's CssInclude and JsInclude.

**Comment by meetmandeep** [comment, marked as answer]
Use the header tags options available in Settings > SEO > Advanced to add tags to HEAD section. There's no option to add site level scripts at bottom of the page. I would create an issue for that as a feature request. 

On page level, you can add it to a custom block and place towards the bottom of the page.

**Reply by X3Technology** [reply]
I like this suggestion as a work-around.  I put a Custom Code block in my Custom Footer that goes on every page.  In this Custom Code block I just have this:
`<script src="/portals/_default/vThemes/Incline/assets/js/navbar.js"></script>`
which is just some javascript to detect scroll position and change the background color of the nav section.  This seems to work pretty well.

I put the CSS link in the SEO section like you said and that also seems to work.
